### PR TITLE
[Task003]create Movie Screen

### PR DESCRIPTION
**Pull request by ando-andy**
## Related Tickets

- [#Ticket ID](https://docs.google.com/spreadsheets/d/1F5V2sIUMGej3Wrp8qYRscf0KaVSlccBmqkRx49_hUVo/edit#gid=935273790)

Create UI for Movies screen
 - List movies now playing using CollectionView
 - List popular movies using CollectionView
 - Title label
 - Search button

Create a UI action
 - Click movie cell to view detail of the selected movie
 - Click the Search button to go to Search screen
 - Click More to view all list

Fetch data from API and display to the UI

## What's this PR do ?
Init base API and models 

## Library
*(List gem, library third party add new)*
Alamofire:  https://github.com/Alamofire/Alamofire
WenchaoD / FSPagerView:  https://github.com/WenchaoD/FSPagerView

## Impacted Areas in Application
*(List features, api, models or services that this PR will affect)*

API document: https://developers.themoviedb.org/3/authentication/create-session

## Checklist
- [x] It was tested in local success ?
- [x] Updated rake task, environment variable at [here](https://???)
- [x] Fill link PR into ticket and the opposite
- [x] Note reason, scope of influence, solution into ticket
- [x] Validate UI/Model/API

## Notes
*(Other notes)*

### CODE REVIEW

File: MovieDBTests/Scenes/App/AppNavigatorMock.swift [hunk 1]
```diff
@@ -1,4 +1,7 @@
+<<<<<<< HEAD
```
Comment (tgdong2296) [inline MovieDBTests/Scenes/App/AppNavigatorMock.swift]: this conflig, please fix it ✌️

File: MovieDBTests/Scenes/App/AppUseCaseMock.swift [hunk 2]
```diff
@@ -1,4 +1,7 @@
+<<<<<<< HEAD
```
Comment (tgdong2296) [inline MovieDBTests/Scenes/App/AppUseCaseMock.swift]: conflict here

File: MovieDBTests/Scenes/App/AppViewModelTests.swift [hunk 3]
```diff
@@ -1,4 +1,7 @@
+<<<<<<< HEAD
```
Comment (tgdong2296) [inline MovieDBTests/Scenes/App/AppViewModelTests.swift]: conflict here

File: MovieDBTests/Scenes/Main/MainNavigatorMock.swift [hunk 4]
```diff
@@ -1,4 +1,7 @@
+<<<<<<< HEAD
```
Comment (tgdong2296) [inline MovieDBTests/Scenes/Main/MainNavigatorMock.swift]: conflict here

File: MovieDBTests/Scenes/Main/MainUseCaseMock.swift [hunk 5]
```diff
@@ -1,4 +1,7 @@
+<<<<<<< HEAD
```
Comment (tgdong2296) [inline MovieDBTests/Scenes/Main/MainUseCaseMock.swift]: conlifct here

File: MovieDBTests/Scenes/Main/MainViewControllerTests.swift [hunk 6]
```diff
@@ -1,4 +1,7 @@
+<<<<<<< HEAD
```
Comment (tgdong2296) [inline MovieDBTests/Scenes/Main/MainViewControllerTests.swift]: conflict here

File: MovieDBTests/Scenes/Main/MainViewModelTests.swift [hunk 7]
```diff
@@ -1,4 +1,7 @@
+<<<<<<< HEAD
```
Comment (tgdong2296) [inline MovieDBTests/Scenes/Main/MainViewModelTests.swift]: conlifct here

File: MovieDBTests/Scenes/Movie/MovieNavigatorMock.swift [hunk 8]
```diff
@@ -1,4 +1,7 @@
+<<<<<<< HEAD
```
Comment (tgdong2296) [inline MovieDBTests/Scenes/Movie/MovieNavigatorMock.swift]: conlifct here

File: MovieDBTests/Scenes/Movie/MovieViewControllerTests.swift [hunk 9]
```diff
@@ -1,4 +1,7 @@
+<<<<<<< HEAD
```
Comment (tgdong2296) [inline MovieDBTests/Scenes/Movie/MovieViewControllerTests.swift]: conlifct here

File: MovieDBTests/Scenes/Movie/MovieViewModelTests.swift [hunk 10]
```diff
@@ -1,4 +1,7 @@
+<<<<<<< HEAD
```
Comment (tgdong2296) [inline MovieDBTests/Scenes/Movie/MovieViewModelTests.swift]: conlifct here

File: MovieDBTests/Scenes/App/AppNavigatorMock.swift (hunk 1 above)
Comment (ando-andy) [reply MovieDBTests/Scenes/App/AppNavigatorMock.swift]: FIxed.🙇‍♂️

**Pull request merged**